packages/generate/src/builders.ts

```diff
@@ -849,7 +851,9 @@ export class DirBuilder {
       let oldContents = "";
       try {
         oldContents = await readFileUtf8(filePath);
-      } catch {}
+      } catch (err) {
+        console.error("Error occurred while reading file", filePath, err);
```

Comment:
Similar to the last comment, let's not add side effects in this linting PR. I think we can maintain the same behavior here by removing the assignment of `oldContents` from outside of the try catch and assign the empty string in the catch block.

```ts
let oldContents;
try {
  oldContents = await readFileUtf8(filepath);
} catch {
  oldContents = "";
}
```

Or even using `then` to avoid the awkward try catch here:

```ts
const oldContents = await readFileUtf8(filepath)
  .then((content) => content)
  .catch(() => "");
```